pkg/cri/config/config_test.go

```diff
@@ -141,6 +141,13 @@ func TestValidateConfig(t *testing.T) {
 			},
 			expectedErr: "invalid `drain_exec_sync_io_timeout`",
 		},
+		"enable unprivileged_icmp or unprivileged_port, but kernel verion is smaller than 4.11": {
```

Comment:
Error: "`default_runtime_name` is empty" does not contain "unprivileged_icmp and unprivileged_port are not allowed to be enabled in kernel versions smaller than 4.11"
Test: TestValidateConfig/enable_unprivileged_icmp_or_unprivileged_port,_but_kernel_verion_is_smaller_than_4.11

Reply:
Thank you for your patient review.